### ISSUE TITLE
Can't preview the highcharts.

### ISSUE THREAD

**Issue by SUN08042HanYue** [hupili/datavis.studio#24]
![屏幕截图(106)](https://user-images.githubusercontent.com/47049043/55301430-2deb2900-546f-11e9-8025-4736a9f0119e.png)

## I don't know why but I can't preview any highcharts in Visual Studio Code. (I put the document in my github document. Is there something wrong?)

I do not know what to do next. Please help me.

**Comment by hupili** [comment]
@SUN08042HanYue , please give following information:

* The GitHub URL of the HTML file
* Can you do live preview with other pages? e.g. a simplest HTML containing only head/body/p tags
* When you "launch on browser", do you see any error messages in Google Developer Console?

**Comment by SUN08042HanYue** [comment]
I can't do any live preview with the pages in Highcharts-7.0.3 but I can do live preview with the pages of my homework. However I can see the chart on Google website. Don't know the reason.

**Comment by hupili** [comment]
> I can't do any live preview with the pages in Highcharts-7.0.3

when this happen, what is the error message?

> When you "launch on browser", do you see any error messages in Google Developer Console?

**Comment by SUN08042HanYue** [comment]
When I "launch on browser", the chart normally shows on the website and there is no error messages in Google Developer Console. I just can's preview it in Visual Studio Code.

**Comment by SUN08042HanYue** [comment]
Maybe there is a connection? Because my images can preview in Visual Studio Code but cannot show on the website...

**Comment by hupili** [comment]
Can you try to quit VSCode and do a clean-start?

* Launch VSCode
* Drag entire folder into the VSCode (sense of "project" for VSCode)
* Enter the HTML page
* cmd-shift-p: type "launch ..." / "preview ...". 

Please see if that works.

**Comment by SUN08042HanYue** [comment]
Yes, I have tried to close the folder and drag the folder into the VSCode many times but it seems like all things are same...

**Comment by hupili** [comment]
@SUN08042HanYue if you are in a hurry, you can use our Lab computers to finish the homework first. Those computers have consistent environment with our class demo. Then you and our TA have more time, we can troubleshoot each specific device/ OS.

**Comment by SUN08042HanYue** [comment]
ok thanks!
